graviti/manager/dataset.py

```diff
@@ -56,18 +84,28 @@ def __init__(
         name: str,
         *,
         alias: str,
+        default_branch: str,
         commit_id: str,
-        branch_name: Optional[str] = None,
+        created_at: str,
+        updated_at: str,
+        owner: str,
+        is_public: bool,
+        config: str,
+        branch: Optional[str] = None,
```

Comment:
can we remove `branch` here just use `default_branch`?

Reply:
branch is useful when we checkout to other branch

Reply:
can we just set `branch = default_branch` in `__init__` and remove the `branch` as input?